<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MazeWindow</class>
 <widget class="QWidget" name="MazeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Maze</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,2,2,2,1" columnstretch="1,5">
   <item row="0" column="0">
    <widget class="QPushButton" name="Button_Back">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="20,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QWidget" name="Content_Maze" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="Button_AutoStop">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_StepBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Step Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_PlayPause">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button_StepForward">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Step Forward</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="Button_ObstaclePoint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ObstaclePoint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_ClearPoint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ClearPoint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_StartPoint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>StartPoint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_FinishPoint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>FinishPoint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_CompleteEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>CompleteEdit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="Button_ViewMap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ViewMap</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_ClearMap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ClearMap</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
